src/services/http/utils/passport.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+const passport = require('passport')
+const JwtStrategy = require('passport-jwt').Strategy
+const ExtractJwt = require('passport-jwt').ExtractJwt
+const { jwt } = require('../../../config/vars')
+const { fetchAdmin } = require('../../storage/handler')
+
+/**
+ * Options for handling JWT requests
+ */
+let opts = {
```

Comment:
I found you use a lot of `let` instead of `const`, could you let me know the reason?

Reply:
In most situations, I delegate configurations so separate files and use const to make sure they are unmutable, as such this would be better being pulled and safer(you can see examples of that in the index.js file).  The main config file is a const and as such I could easily pass that to the function instead